src/siaskydev.test.ts

```diff
@@ -29,4 +31,48 @@ describe.skip("siasky.dev end to end", () => {
     const buf = Buffer.from(parts[1], "base64");
     expect(buf.toString("ascii")).toEqual("thisistext");
   });
+
+  it("support rev number update higher than 255", async () => {
+    // create a random user
+    const rand = randomNumber(0, 1e6);
+    const user = new User(`john.doe+${rand}@gmail.com`, "test1234");
```

Comment:
```suggestion
    const user = new User("john.doe@gmail.com", "test1234");
```
you don't need random number there, it's isolated test case

Reply:
It doesn't really matter much if I randomize the user or the file id right? It's not really isolated either because the SkynetClient is pointed at `siasky.dev`. If anything I'd rather remove this test file altogether if it's an issue keeping it in.  I only wrote these functions to be able to quickly verify the working on siasky.dev instead of relying on the mock adapter.

Reply:
I need to randomize either the user or the file id.

Reply:
Right, I did not realise this is an e2e test. We should probably move it to a separate suite so it's not confused with the unit tests.